docs/running.md

```diff
@@ -30,11 +30,15 @@ Here we assume you have two data sources which you wish to analyse together:
 > As the global dataset grows, subsampling becomes important.
 We will write guidance for subsampling in a future page, but you can investigate the `Snakefile_Regions` to see how we perform subsampling for our regional builds.
 
-#### Global data
+#### Obtaining global data through GISAID
 
 For the nextstrain.org analyses we use data obtained through [GISAID](https//gisaid.org).
-Please see there for how you may obtain those genomic data for your own analysis as the terms of data sharing prevent us making the sequence data publicly available.
-Included in this repository is [a curated list of metadata](../data/metadata.tsv) associated with those sequences.
+
+Once you have looged into [GISAID's EpiCoV site](https://www.epicov.org/epi3/cfrontend) click "partner downloads" to download the (compressed) metadata & sequences.
```

Comment:
Typo in "looged".

Reply:
Remove link to https://www.epicov.org/epi3/cfrontend. Just use "Once you have logged into GISAID's EpiCoV site..."

Reply:
👍 done